parsing/src/recipes.ts

```diff
@@ -118,8 +118,8 @@ function getRecipes(
                 // get the power from the recipe instead of the building
                 lowPower = Number(recipe.mVariablePowerConsumptionConstant);
                 highPower = Number(recipe.mVariablePowerConsumptionFactor);
-                // calculate the average power
-                if (lowPower && highPower) {
+                // calculate the average power: Note that because low power can be 0, (and often is), we can't use truthy checks to validate these values
+                if (lowPower != null && highPower != null) {
```

Comment:
```suggestion
                if (lowPower !== null && highPower !== null) {
```
No loose typing here please :P